src/python/pants/backend/jvm/targets/jar_dependency.py

```diff
@@ -51,16 +111,34 @@ def __new__(cls, org, name, rev=None, force=False, ext=None, url=None, apidocs=N
       the dependency itself should be downloaded and placed on the classpath
     :param list excludes: Transitive dependencies of this jar to exclude.
     :type excludes: list of :class:`pants.backend.jvm.targets.exclude.Exclude`
+    :param string base_path: absolute base path that a relative file url is based from.
```

Comment:
Yea, I'm ok with adding this parameter.

Between the lines of my last comment, I was suggesting adding public documentation for this new feature that will show up here: http://www.pantsbuild.org/build_dictionary.html